src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -20,21 +30,68 @@ test.describe("Buzz Page", () => {
 
   test.afterEach(async ({ page }) => {
     buzzPage = new BuzzPage(page);
-    await buzzPage.deleteTheNewestPost()
+    await buzzPage.deleteTheNewestPost(false)
   });
 
   test("Post should be shared", async ({ page }) => {
+    const randomTitle = generateRandomString()
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePost(filePath);
+    await buzzPage.sharePost(filePath, randomTitle);
```

Comment:
You share a picture in this test, but later on, you don't assert that the post contains a picture. I think you can use bodyTextPost to check if one of the child elements is an image.
![Screenshot 2023-10-10 at 21 59 20](https://github.com/tooploox/orangehrm-qa-playwright-practice/assets/60622255/93bfab02-c6ab-401a-b7cd-c2473ec7369a)

Reply:
will add it

Reply:
i will just check if chidren appears by method --> await expect(buzzPage.photo).toHaveCount(1)

Reply:
This will not work, because it checks for photos on the whole page - not photo in the post you just added. 
I was thinking about having an selector that checks for: 
"orangehrm-buzz-post-body" class - that contains "randomTitle" text in one of the child elements and also contains "orangehrm-buzz-photos" class in one of the child elements. This would ensure that the post you created has the expected text in one of the child elements and also contains a picture container.